addons/mail/models/mail_mail.py

```diff
@@ -508,6 +540,10 @@ def send(self, auto_commit=False, raise_exception=False):
 
     def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
         IrMailServer = self.env['ir.mail_server']
+        # Only retrieve recipient followers of the mails if needed
+        records_followers_set = self.env['mail.followers']._get_mail_followers(self.ids) if self.filtered(
```

Comment:
As stated above, we could directly check here models that have the unfollow mechanism activated, to avoid fetching useless information :)

Reply:
You mean “_partner_unfollow_enabled” attribute ? But even if that attribute isn’t present or False, if the recipients is an internal user, the unfollow link is inserted. Moreover that call returns in one go all follower information for all mail to be sent. So why limit it by model ?